apps/nextra/pages/en/build/indexer/indexer-reference.mdx

```diff
@@ -0,0 +1,395 @@
+| registered_address       | String  |             | This is an Aptos account address registered to the domain. Addresses must be 66 characters long, and so may be 0 padded. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a" |
+| subdomain                | String  |             | Any subdomain part of the domain name. Ex. "sub.example"                                                                            |
+| token_name               | String  |             | The name of the token associated with this domain. Ex. "ExampleToken"                                                               |
+| token_standard           | String  |             | Aptos standard that the collection adheres to. Ex. "v1"                                                                             |
+
+### `current_ans_lookup_v2`
+
+This table maps tokens, standards, and addresses to human readable names.
+
+| Field                    | Type    | Primary Key | Description                                                                                                                         |
+| ------------------------ | ------- | ----------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| domain                   | String  | Yes         | The domain associated with this Aptos name. Ex. "example.crypto"                                                                    |
+| expiration_timestamp     | String  |             | Timestamp when the domain registration expires. Ex. "2024-04-17T02:14:25.68771"                                                     |
+| is_deleted               | Boolean |             | Indicates whether the domain registration has been deleted. Ex. False                                                               |
+| last_transaction_version | bigint  |             | The version number of the last transaction involving this domain. Ex. 5000000                                                       |
+| registered_address       | String  |             | This is an Aptos account address registered to the domain. Addresses must be 66 characters long, and so may be 0 padded. Ex. "0x50bc83f01d48ab3b9c00048542332201ab9cbbea61bda5f48bf81dc506caa78a" |
+| subdomain                | String  | Yes         | Any subdomain part of the domain name. Ex. "sub.example"                                                                            |
+| token_name               | String  |             | The name of the token associated with this domain. Ex. "ExampleToken"                                                               |
+| token_standard           | String  | Yes         | Aptos standard that the collection adheres to. Ex. "v1"                                                                             |
+
+<br/>
+# Deprecated Tables
```

Comment:
some of the tables below are not planned for deprecation. https://www.notion.so/aptoslabs/Deprecated-Tables-f28de120b5474484ae9b6063ba7a4bdf?pvs=4 I added dates and notes. We will have finalized dates for some of tables that will be completely removed from our AlloyDB.